tests/cucumber/steps/steps.js

```diff
@@ -121,6 +121,10 @@ module.exports = function getSteps(options) {
     steps.then[name] = fn;
   }
 
+  // We need to import inside the steps export so that the browser tests have the dependency imported
+  // eslint-disable-next-line global-require
+  const JSONBig = require('json-bigint');
```

Comment:
I'm not sure if this will work. However we do have the `tests/cucumber/browser/test.js` file which can be used to populate imports for browser tests.

Adding something like this to that file should work (we do the same for the `assert` package):

```javascript
const JSONBig = require('json-bigint');
window.JSONBig = JSONBig;
```